src/get-started/install/_windows-desktop-setup.md

```diff
@@ -26,4 +18,4 @@ you need the following in addition to the Flutter SDK:
 For more information, see [Desktop support for Flutter][]
 
 [Desktop support for Flutter]: {{site.url}}/desktop
-[Visual Studio 2022]: https://visualstudio.microsoft.com/downloads/
\ No newline at end of file
+[Visual Studio 2022]: https://visualstudio.microsoft.com/downloads/
```

Comment:
I'm curious what changed here. I can't see the delta

Reply:
I think GitHub's editor either added or removed a trailing newline.